pkg/integrations/install/install.go

```diff
@@ -31,6 +31,7 @@ import (
 	_ "github.com/grafana/agent/pkg/integrations/snowflake_exporter"     // register snowflake_exporter
 	_ "github.com/grafana/agent/pkg/integrations/squid_exporter"         // register squid_exporter
 	_ "github.com/grafana/agent/pkg/integrations/statsd_exporter"        // register statsd_exporter
+	_ "github.com/grafana/agent/pkg/integrations/vmware_exporter"        // register vmware_exporter
```

Comment:
We also need to change the import path of the v2 integration below on line 47.

But do we know if there's a reason why the integration was v2-only?

Reply:
Not really, it was originally added my @rlankfo